src/components/coin.ts

```diff
@@ -68,7 +68,7 @@ export interface UserCoinBonus {
   last_granted: Date;
 }
 
-export const getCoinBalanceByUserId = async (userId: string): Promise<number> => {
+export const getCoinBalanceByUserId = async (userId: string): Promise<number | undefined> => {
```

Comment:
will we ever have undefined?

Reply:
We won't, we just did because I had try catches that I removed earlier